16_bg_slider/app.js

```diff
@@ -0,0 +1,34 @@
+const btnPrev = document.querySelector(".prev")
+const btnNext = document.querySelector(".next")
+
+var slideIndex = 1;
+showSlides(slideIndex);
+
+function nextImage(n) {
+    showSlides(slideIndex += n)
```

Comment:
Here's a tip to help you debug the code. Use `console.log()`. A lot. It's one of your BFFs when it comes to understanding JS.

If you do `console.log(n)` before this line, you'll see that `n` actually is an event because it's called through the event listener (click).

Another tip for you to figure out this challenge. Try to draw the "data flow" in a piece of paper with a pen. And try to draw what's needed to make this work. That will help you to "design" better code :)